cost-analyzer/templates/_helpers.tpl

```diff
@@ -54,8 +54,10 @@ Kubecost 2.0 preconditions
   {{- fail "Kubecost no longer includes Thanos by default. Please see https://docs.kubecost.com/install-and-configure/install/thanos for more information." -}}
   {{- end -}}
 
-  {{- if (.Values.federatedETL).primaryCluster -}}
```

Comment:
superseded by https://github.com/kubecost/cost-analyzer-helm-chart/blob/f34a9e00ce713480369fb1eb597f1d3d019d5c5d/cost-analyzer/templates/_helpers.tpl#L7

Reply:
Nice! I've validated that there are no other references to `.Values.federatedETL.primaryCluster` on the `develop` branch. So this should smoothly transition FedETL users over, without any unintended FedETL side-effects.